go.sum

```diff
@@ -316,6 +318,8 @@ github.com/golang-jwt/jwt v3.2.1+incompatible h1:73Z+4BJcrTC+KczS6WvTPvRGOp1WmfE
 github.com/golang-jwt/jwt v3.2.1+incompatible/go.mod h1:8pz2t5EyA70fFQQSrl6XZXzqecmYZeUEB8OUGHkxJ+I=
```

Comment:
Can we clean up the `go.sum` to remove the older versions if they are not being referenced by any dependency?

Reply:
cleaned up `go.sum`